subgraphs/playback/src/resolvers/Subscription.ts

```diff
@@ -1,22 +1,22 @@
-import { SubscriptionResolvers } from './types';
-import { Spotify } from '../dataSources/spotify.types';
-import { TOPICS } from '../constants';
-import { map, distinctUntilChanged } from 'rxjs';
-import { equal } from '@wry/equality';
-import { GraphQLResolveInfo } from 'graphql';
-import { omit } from 'lodash';
-import { PartialDeep } from 'type-fest';
-import { selectsField } from '../utils/graphql';
-import { createPlaybackStateObservable } from '../observables';
+import { SubscriptionResolvers } from "../__generated__/resolvers-types";
+import { Spotify } from "../dataSources/spotify.types";
+import { TOPICS } from "../utils/constants";
+import { map, distinctUntilChanged } from "rxjs";
+import { equal } from "@wry/equality";
+import { GraphQLResolveInfo } from "graphql";
+import { omit } from "lodash";
+import { PartialDeep } from "type-fest";
+import { selectsField } from "../utils/graphql";
+import { createPlaybackStateObservable } from "../observables";
```

Comment:
Looks like the prettier config changed or something. Mind if we use single quotes for prettier config?